docs/dev-guide/src/architecture/runtime/runtime.md

```diff
@@ -41,9 +41,11 @@ These pallets are all part of substrate and aren't maintained by the creditcoin
 - [Balances](https://paritytech.github.io/polkadot-sdk/master/pallet_balances/index.html)
 - [FRAME System](https://paritytech.github.io/polkadot-sdk/master/frame_system/index.html)
 - [Scheduler](https://paritytech.github.io/polkadot-sdk/master/pallet_scheduler/index.html)
+- [Staking](https://paritytech.github.io/polkadot-sdk/master/pallet_staking/index.html)
 - [Sudo](https://paritytech.github.io/polkadot-sdk/master/pallet_sudo/index.html)
 - [Timestamp](https://paritytech.github.io/polkadot-sdk/master/pallet_timestamp/index.html)
 - [Transaction Payment](https://paritytech.github.io/polkadot-sdk/master/pallet_transaction_payment/index.html)
+- [Utility](https://paritytech.github.io/polkadot-sdk/master/pallet_utility/index.html)
```

Comment:
All of the above URLs should actually be links under https://github.com/gluwa/substrate

Reply:
These are Github.io links to the built & deployed Rust docs. We don't have them and the docs state that these external pallets are not maintained by the Creditcoin devs.

I think we should keep the links to the upstream docs.

Reply:
I see now. Missed the `.io` pert previously.